superset/__init__.py

```diff
@@ -39,6 +40,41 @@ def cast_form_data(form_data):
         d[k] = v
     return d
 
+def cast_filter_data(form_data):
```

Comment:
I think this function should be defined in the migration script as there will be no use for it beyond the migration.